### DISCUSSION TITLE
Mapping AccountName

### DISCUSSION THREAD

**Discussion by micooro** [YetiForceCompany/YetiForceCRM#15530] [How to] [answered]
Hi to all.
I tried since a one month to map "account name" from Account Module to field "account" in "Sales Invoice" module.
If i try to add record into my account, the field "account" in "Sales Invoice" module is filled, but if try throught button "generate record" on top, the field is not filled. I tryed to set in admin panel----Software configuration----Standard modules----Mapping, but i cannot map this field. How can i do?
Thanks for your suggest..

Mike

**Comment by vovpff** [comment, marked as answer]
Are you tried this?

![image](https://user-images.githubusercontent.com/8545791/145401367-c9829c48-32ea-409f-aae2-38dd85764b27.png)

**Reply by micooro** [reply]
Wow..................you're magician, you saved me!!!! I was going crazy. THANK you very much, as it was a long time since I could not find a solution and I imagined absurd workarounds.
Very very thanks.

Mike
